### PR TITLE
Makes prosthetics editable in appearance editor

### PR DESCRIPTION
**Pull request by RustingWithYou**
The appearance editor can now change organs and limbs into prosthetic/nymph/amputated versions.
As ghostroles can now edit their own limbs, the Tarwa Conglomerate ghostroles no longer have random nymph limbs.
Height and speech bubbles no longer need the user to be able to change their species to modify them

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

**Comment by dessysalta** [comment]
> The appearance editor can now change organs and limbs into prosthetic/nymph/amputated versions.

i love you

**Comment by Sniblet** [comment]
One step closer to offships being real people

**Comment by RustingWithYou** [comment]
> One step closer to outsiders being real people I've noticed that only antags got the ability to edit height, and not ghostspawns. I guess they have separate appearance editors? I can't read so I don't know why this happens. I hope this works for both!

The reason for this is that height is in the species window which only shows up if the appearance editor lets you change your species. I might see about changing that in this PR too, actually

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/appearance.dm [hunk 1]
```diff
@@ -143,6 +143,69 @@
...
+	switch(species.name)
+		if(SPECIES_HUMAN, SPECIES_HUMAN_OFFWORLD)
+			valid_prosthetics += PROSTHETIC_SYNTHSKIN
+		if(SPECIES_UNATHI)
+			valid_prosthetics += PROSTHETIC_AUTAKH
+		if(SPECIES_TAJARA, SPECIES_TAJARA_MSAI, SPECIES_TAJARA_ZHAN)
+			valid_prosthetics += PROSTHETIC_TESLA
+		if(SPECIES_VAURCA_WORKER, SPECIES_VAURCA_WARRIOR)
+			valid_prosthetics += PROSTHETIC_VAURCA
+	return valid_prosthetics
```
[the remark this answers is not in the document]
Comment (RustingWithYou) [reply code/modules/mob/living/carbon/human/appearance.dm]: Moved to species level

**Review by NonQueueingMatt — APPROVED** [review]

**Review by DreamySkrell — APPROVED** [review]

**Pull request merged**